yt/data_objects/tests/test_firefly.py

```diff
@@ -1,6 +1,12 @@
 import tempfile
+from contextlib import nullcontext as dnr
```

Comment:
why this alias ? `nullcontext` may be longer to type but it's much clearer IMO

Reply:
Mostly for line length and to match the [pytest doc](https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising).
I've changed `dnr` to `does_not_raise` to be clearer. Also, I think using `does_not_raise()` is a little clearer than `nullcontext()` in the `@pytest.mark.parametrize` portion.

Reply:
seems reasonable, thanks